components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/util/IdPManagementConstants.java

```diff
@@ -30,6 +30,27 @@
     public static final String MULTI_VALUED_PROPERT_IDENTIFIER_PATTERN = ".*\\" + MULTI_VALUED_PROPERTY_CHARACTER +
             "[0-9]+";
     public static final String META_DATA = "meta_data";
+    public static final int DEFAULT_RESULTS_PER_PAGE = 15;
+    public static final int MAXIMUM_LIMIT_PER_PAGE = 100;
+    public static final String DEFAULT_SORT_BY = "ID";
+    public static final String ITEMS_PER_PAGE_PROPERTY = "ItemsPerPage";
+    public static final String DESC_SORT_ORDER = "DESC";
+    public static final String DEFAULT_SORT_ORDER = "ASC";
+    public static final String EMPTY_STRING = "";
+    public static final String NAME = "NAME";
+    public static final String IDP_NAME = "IdpName";
```

Comment:
is this constant use to track the param coming from request? For idpName, request will come as 'name'